packages/mdc-chips/README.md

```diff
@@ -204,6 +228,10 @@ Property | Value Type | Description
 --- | --- | ---
 `chips` | Array<`MDCChip`> | An array of the `MDCChip` objects that represent chips in the set
 
+## Usage within Web Frameworks
+
+If you are using a JavaScript framework, such as React or Angular, you can create a <COMPONENT_NAME> for your framework. Depending on your needs, you can use the _Simple Approach: Wrapping MDC Web Vanilla Components_, or the _Advanced Approach: Using Foundations and Adapters_. Please follow the instructions [here](../integrating-into-frameworks.md).
```

Comment:
<COMPONENT_NAME> -> Chip Set and Chip?

Also, the "here" link needs a different relative path (see #2651)